lib/pavilion/test_config/resolver.py

```diff
@@ -393,6 +397,14 @@ def load_raw_configs(self, tests, host, modes):
                         "Test suite '{}' raised a type error, but that "
                         "should never happen. {}".format(test_suite_path, err))
 
+                # Check test compatibility with current pavilion version
+                comp_error = self.check_suite_version_compatibility(test_suite_cfg)
```

Comment:
You should only check this if validation fails on the config. The best place to do that is during validation in 'resolve_inheritance', where it can be done on a test-by-test basis.